versioned_docs/version-2.11.x/admin-api-topics.md

```diff
@@ -522,6 +522,28 @@ admin.topics().peekMessages(topic, subName, numMessages);
 
 </TabItem>
 
+</Tabs>
+````
+### Peek messages with Offset
+
+You can also start to peek number of messages for a specific subscription of a given topic from a particular offset position in the following ways.
```

Comment:
`start`? 
Maybe 
You can peek a number of messages, for a specific subscription of a topic, from a given offset position, in the following ways: